tests/phpunit/tests/option/wpPrimeNetworkOptionCaches.php

```diff
@@ -0,0 +1,306 @@
+				$cache_key = $option;
+			}
+			$cache_keys[ $option ] = $cache_key;
+		}
+
+		/*
+		 * Set values for the options,
+		 * clear the cache for the options,
+		 * check options are not in cache initially.
+		 */
+		foreach ( $cache_keys as $option => $cache_key ) {
+			update_network_option( $network_id, $option, "value_$option" );
+			wp_cache_delete( $cache_key, $cache_group );
+			$this->assertFalse( wp_cache_get( $cache_key, $cache_group ), "$option was not deleted from the cache." );
+		}
+
+		// Call the wp_prime_option_caches function to prime the options.
+		wp_prime_network_option_caches( $network_id, $options_to_prime );
+
+		// Store the initial database query count.
+		$initial_query_count = get_num_queries();
+
+		// Check that options are only in the 'options' or 'site-options' cache group.
+		foreach ( $cache_keys as $option => $cache_key ) {
+			$this->assertSame( "value_$option", wp_cache_get( $cache_key, $cache_group ), "$option cache is not primed" );
+			$this->assertSame(
+				"value_$option",
+				get_network_option( $network_id, $option ),
+				"$option has not been loaded"
+			);
+		}
+
+		// Ensure no additional database queries were made.
+		$this->assertSame(
+			$initial_query_count,
+			get_num_queries(),
+			'Additional database queries were made.'
+		);
+	}
+
+	/**
+	 * Tests that wp_prime_network_option_caches() is run twice
+	 *
+	 * @ticket 61053
+	 */
+	public function test_wp_prime_network_option_caches_run_twice() {
+		// Create some options to prime.
+		$network_id = get_current_network_id();
+		if ( is_multisite() ) {
+			$cache_group = 'site-options';
+		} else {
+			$cache_group = 'options';
+		}
+
+		// Create some options to prime.
+		$options_to_prime = array(
+			'option1',
+			'option2',
+			'option3',
+		);
+
+		$cache_keys = array();
+		foreach ( $options_to_prime as $option ) {
+			if ( is_multisite() ) {
+				$cache_key = "$network_id:$option";
+			} else {
+				$cache_key = $option;
+			}
+			$cache_keys[ $option ] = $cache_key;
+		}
+
+		/*
+		 * Set values for the options,
+		 * clear the cache for the options,
+		 * check options are not in cache initially.
+		 */
+		foreach ( $cache_keys as $option => $cache_key ) {
+			update_network_option( $network_id, $option, "value_$option" );
+			wp_cache_delete( $cache_key, $cache_group );
+			$this->assertFalse( wp_cache_get( $cache_key, $cache_group ), "$option was not deleted from the cache." );
+		}
+
+		// Call the wp_prime_option_caches function to prime the options.
+		wp_prime_network_option_caches( $network_id, $options_to_prime );
+
+		// Store the initial database query count.
+		$initial_query_count = get_num_queries();
+
+		// Call the wp_prime_option_caches function second time
+		wp_prime_network_option_caches( $network_id, $options_to_prime );
+
+		// Ensure no additional database queries were made.
+		$this->assertSame(
+			$initial_query_count,
+			get_num_queries(),
+			'Additional database queries were made.'
+		);
+	}
+	/**
```

Comment:
I suspect GH got confused with the empty lines.

```suggestion
	}

	/**
```